### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by terrikwak**
I updated the Korean readme. Please take a look

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,35 @@
...
+
+**Workflow summary**
+
+- Submitted translations get approved by our maintainers.
+- The approved translations will be pushed to the `l10n` branch of `klaytn-docs`. The `dev` branch of this repo mirrors the `ko/docs` subfolder of `klaytn-docs:l10n` periodically. 
+- Our maintainers will merge the changes to the `master` branch, which will be reflected at https://ko.docs.klaytn.com. 
+
+Note: **Do NOT push to `klaytn-docs:l10n` directly.** It will be overwritten by Crowdin and the changes will get lost.
+
+To start translating, follow the link - https://crwd.in/klaytn-docs.
```
Comment (kjhman21) [inline README.md:25]: Seems that link is broken?

Comment (terrikwak) [inline README.md:25]:
```suggestion
To start translating, follow the [link](https://crowdin.com/project/klaytn-docs/invite?d=85n6j4j675t683c3h463e3t6c3q4r48383d3)
```

Comment (terrikwak) [reply README.md]: @kjhman21  I fixed the link! Can you approve now? 😃

**Review by kjhman21 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,13 +1,35 @@
...
+
+## Want to Contribute to Klaytn Docs? <a id="want-to-contribute"></a>
+
+You can also contribute to the source text in English in [Klaytn Docs](https://github.com/klaytn/klaytn-docs).
+
+You can do that by creating a pull request on GitHub. For more details, please refer to [Contribution Guide](https://github.com/klaytn/klaytn-docs/wiki#contributing).
+
+## Need Help? <a href="#need-help" id="need-help"></a>
+
+If you have any questions, please visit our [Gitter channel](https://gitter.im/klaytn/klaytn-docs?utm_source=share-link&utm_medium=link&utm_campaign=share-link), [Klaytn Developers Forum](https://forum.klaytn.com/) and [Discord channel](https://discord.gg/mWsHFqN5Zf).
```
Comment (kjhman21) [inline README.md:35]: @neoofklaytn Do we have any analytics connected with gitter? It seems we have utm tags in the link.

Comment (neoofklaytn) [reply README.md:35]: @kjhman21 This is the standard link with pre-configured utm tags, from Gitter. We have not had analytics plugin integrated to the channels and there are plug-in options we could look to integrate on Gitter. Thanks.

**Pull request closed without merging**